### PR TITLE
Temporary fix for 3670

### PR DESCRIPTION
**Pull request by LefterisJP**
Temporary fix for #3670

### CODE REVIEW

File: raiden/ui/app.py [hunk 1]
```diff
@@ -323,6 +323,12 @@ def run_app(
             )
             sys.exit(1)
 
+        services_deployment_data = get_contracts_deployed(
+            chain_id=node_network_id,
+            version=contracts_version,
+            services=True,
+        )
+        services_contracts = services_deployment_data['contracts']
```
Comment (rakanalh) [inline raiden/ui/app.py]: AFAIK, this will fail if you run raiden in production-mode. I might be mistaken though

Comment (LefterisJP) [reply raiden/ui/app.py]: You are right. This interface is ridiculous. It makes no sense. Let me fix this then.

File: raiden/ui/app.py [hunk 2]
```diff
@@ -418,10 +420,11 @@ def run_app(
 
     service_registry = None
     if routing_mode == RoutingMode.PFS:
+        services_contracts = services_deployment_data['contracts']
```
Comment (rakanalh) [inline raiden/ui/app.py]: This means that using PFS in **production** mode will result in "used-before-declaration", no?

Comment (rakanalh) [reply raiden/ui/app.py]: Also, `service_contracts` is set twice. First one would be in line 333

Comment (LefterisJP) [reply raiden/ui/app.py]: Hmm I have removed this in the last commit.

Comment (LefterisJP) [reply raiden/ui/app.py]: As I said this is removed in the last commit, github must have shown you an older state?

**Pull request merged**